vllm/executor/ray_gpu_executor.py

```diff
@@ -206,8 +206,8 @@ def sort_by_driver_then_worker_ip(worker):
                 ray.get(worker.get_node_and_gpu_ids.remote()) \
             ) # type: ignore
 
-        node_workers = defaultdict(list)  # node id -> list of worker ranks
-        node_gpus = defaultdict(list)  # node id -> list of gpu ids
+        node_workers: DefaultDict[str, List[int]] = defaultdict(list)  # node id -> list of worker ranks
+        node_gpus: DefaultDict[str: List[Union[int, str]]] = defaultdict(list)  # node id -> list of gpu ids
```

Comment:
```suggestion
        node_workers = defaultdict[str, list[int]](list)  # node id -> list of worker ranks
        node_gpus = defaultdict[str, list[Union[int, str]]](list)  # node id -> list of gpu ids
```

In Python 3.9+, we can just simply call getitem on the class itself